lib/benchee/formatters/console/run_time.ex

```diff
@@ -6,8 +6,13 @@ defmodule Benchee.Formatters.Console.RunTime do
   these two things into separate modules to avoid confusion.
   """
 
-  alias Benchee.{Statistics, Benchmark.Scenario, Conversion, Formatters.Console.Helpers}
-  alias Benchee.Conversion.{Count, Unit}
+  alias Benchee.{
+    Benchmark.Scenario,
+    Conversion,
+    Conversion.{Count, Unit},
+    Formatters.Console.Helpers,
+    Statistics
+  }
```

Comment:
some of this sorting was made due to credo 0.9.3 but it started crashing... I guess it doesn't hurt. Although our alias style is inconsistent.